server/routes/helpers.ts

```diff
@@ -89,10 +96,32 @@ export function getModelsFromResponses(modelHits: any[]): ModelDict {
   return modelDict;
 }
 
+// Convert the workflow state into a readable/presentable state on frontend
 export function getWorkflowStateFromResponse(
   state: typeof WORKFLOW_STATE | undefined
 ): WORKFLOW_STATE {
   const finalState = state || DEFAULT_NEW_WORKFLOW_STATE_TYPE;
   // @ts-ignore
   return WORKFLOW_STATE[finalState];
 }
+
+// Convert the workflow resources into a readable/presentable state on frontend
+export function getResourcesCreatedFromResponse(
+  resourcesCreated: any[] | undefined
+): WorkflowResource[] {
+  const finalResources = [] as WorkflowResource[];
+  if (resourcesCreated) {
+    resourcesCreated.forEach((backendResource) => {
+      finalResources.push({
+        id: backendResource.resource_id,
+        type:
+          // @ts-ignore
```

Comment:
what's the motivation of adding `@ts-igore` here? does it mean it's possible that `WORKFLOW_RESOURCE_TYPE` cannot handle all cases?

Reply:
See the added enum in `common/interfaces`. The ignore here is because of the type safety issue when referencing an enum with a `string` value instead of `typeof WORKFLOW_RESOURCE_TYPE`. Note I do the same on line 104-105.

When the lookup fails this will propagate that error through `flow_framework_routes_service` correctly and make it visible. We don't want to hide a critical parsing error where the backend and frontend schema become out of sync, so we handle it this way.